universal-application-tool-0.0.1/app/services/program/ProgramService.java

```diff
@@ -54,26 +55,38 @@
   /**
    * Create a new program with an empty block.
    *
-   * @param name a name for this program
-   * @param description the description of what the program provides
+   * @param adminName a name for this program
+   * @param adminDescription the description of what the program provides
    * @return the {@link ProgramDefinition} that was created if succeeded, or a set of errors if
    *     failed
    */
   ErrorAnd<ProgramDefinition, CiviFormError> createProgramDefinition(
-      String name, String description);
+      String adminName,
+      String adminDescription,
+      String defaultDisplayName,
```

Comment:
Suggestion (blocking): would you mind adding this to doctring?

Reply:
Great catch! Added